.github/workflows/go.yml

```
@@ -61,140 +61,140 @@ jobs:
+              run: |
+                echo "Previous Coverage ${{ steps.master-coverage.outputs.coverage }}"
+
+            - name: Current coverage
+              run: |
+                echo "Current Coverage ${{ steps.current-coverage.outputs.coverage }}"
+
+            - name: post coverage report
+              if: github.event_name == 'pull_request'
+              uses: thollander/actions-comment-pull-request@v3
+              with:
+                github-token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)